Specs/Renderer/BuiltinFunctionsSpec.js

```diff
@@ -450,6 +450,7 @@ describe(
 
     it("has czm_unpackFloat", function () {
       var packed = Cartesian4.packFloat(1);
+      packed = Cartesian4.divideByScalar(packed, 255, packed);
```

Comment:
This test was incorrect before but worked by accident. `czm_unpackFloat` expects the packed floats to be in a unsigned normalized U8.